packages/node_modules/@ciscospark/plugin-phone/test/integration/spec/phone.js

```diff
@@ -240,6 +240,69 @@ browserOnly(describe)('plugin-phone', function () {
         }));
     });
 
+    describe('#listActiveCalls()', () => {
+      beforeEach('empty emittedCalls collection', () => {
+        spock.spark.phone.emittedCalls.reset();
+      });
+
+      describe('when no ongoing calls exist', () => {
+        it('should be empty', () =>
+          spock.spark.phone.listActiveCalls()
+            .then((callList) => {
+              assert.isTrue(Array.isArray(callList));
+              assert.equal(callList.length, 0);
+            }));
+      });
+
+      describe('when a ongoing call exists', () => {
+        let connectedCall;
+        beforeEach('create an existing call', () => handleErrorEvent(spock.spark.phone.dial(mccoy.email), (call) => expectCallIncomingEvent(mccoy.spark.phone)
+          .then((mc) => Promise.all([
+            expectConnectedEvent(call),
+            mc.answer()
+          ]))
+          .then(() => {
+            connectedCall = call;
+          })));
+
+        afterEach(() => connectedCall.hangup());
+
+        it('lists the existing call', () => {
+          assert.isDefined(connectedCall);
+          return spock.spark.phone.listActiveCalls()
+            .then((result) => {
+              assert.isTrue(Array.isArray(result));
+              assert.equal(result.length, 1);
+              const activeCall = result[0];
+              assert.equal(activeCall.id, connectedCall.id);
+              assert.equal(activeCall.id, connectedCall.id);
+            });
+        });
+
+        it('adds call to emittedCalls collection', () => {
```

Comment:
Please add missing "the"s to test statements. "adds the call to emitte...", "does not add the call to the emitted...", etc

Reply:
done